README.md

```diff
@@ -1,38 +1,60 @@
 # Truss [![Build Status](https://travis-ci.org/TuneLab/go-truss.svg?branch=master)](https://travis-ci.org/TuneLab/go-truss)
 
-Truss handles the painful parts of services, freeing you to focus on the business logic.
+Truss handles the painful parts of services, freeing you to focus on the
+business logic.
 
-**Please note that Truss is currently pre-release software, and may change drastically with no notice. There is no versioning, no guarantees, no stability at this time. However, if you want to play around, make suggestions, or submit changes, we welcome issues and pull requests!**
+**Please note that Truss is currently pre-release software, and may change
+drastically with no notice. There is no versioning, no guarantees, no stability
+at this time. However, if you want to play around, make suggestions, or submit
+changes, we welcome issues and pull requests!**
 
 ![Everything all the time forever](http://i.imgur.com/FCmSUiQ.png)
 
 ## Install
 
-Currently, there is no binary distribution of Truss, you must install from source.
+Currently, there is no binary distribution of Truss, you must install from
+source.
 
 To install this software, you must:
 
-1. Install protocol buffers version 3 or greater. The easiest way is to download a release from the [github releases](https://github.com/google/protobuf/releases) and add the binary to your `$PATH`. Otherwise [install from source.](https://github.com/google/protobuf/releases)
-2. Of course, install the Go compiler and tools from https://golang.org/. See https://golang.org/doc/install for details.
-3. Install the `protoc-gen-go` and `proto` packages for Go. The simplest way is to run 
+1. Install protocol buffers version 3 or greater. The easiest way is to
+download a release from the [github releases](https://github.com/google/protobuf/releases)
+and add the binary to your `$PATH`. 
+Otherwise [install from source.](https://github.com/google/protobuf/releases)
+2. Of course, install the Go compiler and tools from https://golang.org/.
+See https://golang.org/doc/install for details.
+3. Install the `protoc-gen-go` and `proto` packages for Go. The simplest 
+way is to run 
```

Comment:
Looks like we need to add `go get google.golang.org/genproto/...` into this list of things to do before Truss can be used, yeah? Or, rather, before the output of Truss can be used.